config/schemas/config.yaml

```
@@ -53,9 +53,15 @@ $defs:
     description: Extra arguments passed to a container
     type: array
   fluentdBuffer:
+    title: Fluentd buffer configuration
+    description: Fluentd buffer
```

```suggestion
    description: |-
      Fluentd buffer configuration parameters.
      
      > [!note]
      > See [upstream documentation for reference](https://docs.fluentd.org/configuration/buffer-section#parameters), set keys will be converted from `camelCase` to `snake_case`.
```

I've taken this approach to descriptions:
1. Use `|-` for string blocks to trim trailing whitespace.
2. Treat the text as regular sentences with punctuation.